x-pack/test/functional/apps/maps/blended_vector_layer.js

```diff
@@ -27,7 +27,8 @@ export default function ({ getPageObjects, getService }) {
 
     it('should request documents when zoomed to smaller regions showing less data', async () => {
       const hits = await PageObjects.maps.getHits();
-      expect(hits).to.equal('33');
+      // Allow a range of hits to account for variances in browser window size.
+      expect(parseInt(hits)).to.be.within(30, 40);
```

Comment:
Discussed with @nreese, and he agreed that in this case it would be fine to make this assertion a range of values to account for small variances in screen size.